index.html

```diff
@@ -598,870 +529,53 @@ <h3 id="profile-description-methodology">Methodology</h3>
-              Description</a> conforming to the <a>Core
-              profile</a>:
-          </p>
-          <table class="def">
-            <thead>
-              <tr>
-                <th>keyword</th>
-                <th>type</th>
-                <th>constraint</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr>
-                <td>security</td>
-                <td>string or Array of string</td>
-                <td><code>security</code> at form
-                  level MUST NOT be used.</td>
-              </tr>
-              <tr>
-                <td>scopes</td>
-                <td>string or Array of string</td>
-                <td><code>scopes</code> MUST NOT be
-                  used.</td>
-              </tr>
-            </tbody>
-          </table>
-        </section>
-      </section>
-
-      <!-- Links -->
-
-      <section>
-        <h3 id="links">Links</h3>
-        <section class="ednote">TODO: Consider
-          selecting a defined set from [[RFC8288]].</section>
-
-        <p>
-          The "type" relationship as defined in chapter 6 of
-          [[RFC6903]] is reserved for indicating an instance
-          relationship between a thing and a thing template.
-          The <a>Core Data Model</a> does not put additional
-          constraints or requirements on links. The
-          interpretation of a link is out of scope.
-        </p>
-      </section>
-
-      <!--  Security -->
-
-      <section>
-        <h3 id="security">Security</h3>
-        <p>
-          The <a>Core Data Model</a> defines a subset of the
-          security schemes that MAY be implemented on resource
-          constrained devices. A security scheme MUST be
-          defined at the thing level. The security scheme is
-          applied to the thing as a whole, a thing may adopt
-          multiple security schemes.
-        </p>
-        <p>
-          The set of security schemes supported in the <a>Core
-            Data Model</a> is based on the PlugFest results. To
-          ensure interoperability, a TD consumer, which
-          compliant with the <a>Core Data Model</a> MUST
-          support <strong>all</strong> of the following
-          security schemes:
-        </p>
-
-        <ul>
-          <li>no security</li>
-          <li>Basic Auth</li>
-          <li>Digest</li>
-          <li>Bearer Token</li>
-          <li>Oauth2</li>
-        </ul>
+    <p class="ednote">
+      The name "Core Profile" is still under discussion
+      (see <a href="https://github.com/w3c/wot-profile/issues/21">#21</a>).
+    </p>
 
-        <section>
-          <h4>Recommended Practice</h4>
-          <p>When using the "no security" or "Basic Auth"
-            security schemes it is strongly recommended to
-            use transport layer encryption.</p>
-        </section>
-      </section>
+    <!-- Identifier -->
+    <section id="identifier">
+      <h3>Identifier</h3>
+      <p class="ednote">
```

Comment:
should be in sync with https://github.com/w3c/wot-thing-description/issues/806

Reply:
If `profile` is defined in the Thing Description specification, do you think this section is still needed in the Profile specification? Or should the Profile specification just refer to the relevant section of the Thing Description specification in its introduction for example?

Is anyone working on a PR for https://github.com/w3c/wot-thing-description/issues/806 yet?